docker/private-npm-registry/docker-compose.yml

```diff
@@ -1,12 +1,17 @@
-version: '2.1'
+version: "3"
+
 services:
   verdaccio:
     image: verdaccio/verdaccio:4.6
     container_name: verdaccio-local-registry
     ports:
-      - "4873:4873"
+      - 4873:4873
     volumes:
-      - "./conf:/verdaccio/conf"
-volumes:
-  verdaccio:
-    driver: local
+      - ./conf:/verdaccio/conf
+    networks:
+      - optic
+
+networks:
+  optic:
+    name: optic
```

Comment:
my linux box complained about this field, removing it worked for me

Reply:
that the network didnt exist? `task utility:create-docker-network`

Reply:
worked through this on slack, it was an out of date docker-compose in the ubuntu repos. installing from the docker repo resolved the issue.